x/market/keeper/deal.go

```diff
@@ -135,7 +135,7 @@ func (k Keeper) SellOwnData(ctx sdk.Context, seller sdk.AccAddress, cert types.D
 
 	// Is deal activated
 	if findDeal.GetStatus() != ACTIVE {
-		return sdk.Coin{}, fmt.Errorf("deal is not activated")
+		return sdk.Coin{}, sdkerrors.Wrap(err, "deal is not activated")
```

Comment:
This `err` is an error from `k.GetDeal`.
In keeper func, why don't you define sentinel error and use it? If then, I think you don't have to wrap the sentinel error here.